crates/forge/bin/cmd/bind.rs

```diff
@@ -158,9 +159,29 @@ impl BindArgs {
             .into()
     }
 
+    fn get_alloy_filter(&self) -> Filter {
```

Comment:
is this necessary? The filtering and file finding behavior should be the same

Reply:
Yeah, so `get_filter` returns `ContractFilter` which is specific to `ethers`.  These methods just return the regex that is then used in a unified setting in `get_json_files` to actually get the files.